code/model3.py

```diff
@@ -22,7 +22,7 @@ class Model3(nn.Module, Model):
      representation is obtained by averaging embeddings from pre-trained word2vec model
     """
 
-    def __init__(self, pre_trained=False, load_from=None):
+    def __init__(self, pre_trained=False, load_from=None, hidden_dims=[500]):
```

Comment:
Once model3deeper starts working, can it replace model3?
The hidden dims variable is not being used

Reply:
yes it can